manifests/init.pp

```diff
@@ -90,13 +90,14 @@
 #                                     the puppet agent.
 #                                     type:integer
 #
-# $autosign::                         Enable or disable autosign or change location
-#                                     of autosign.conf or autosign script.  If this
-#                                     is set to a script, make sure that script
-#                                     considers the content of autosign.conf, as it
-#                                     might break Foreman functionality if it
-#                                     doesn't. If this is set to a boolean, it can
-#                                     be cast as a string or a boolean.
+# $autosign::                         If set to a boolean, autosign is enabled or disabled
+#                                     for all incoming requests. Otherwise this has to be
+#                                     set to the full file path of an autosign.conf file or
+#                                     an autosign script. If this is set to a script, make
+#                                     sure that script considers the content of autosign.conf
```

Comment:
If this is a script, should the file still be managed?  Now it's going to have a file { } resource, while before we only set autosign in puppet.conf.